search.yaml

```diff
@@ -0,0 +1,433 @@
+        explode: true
+        schema:
+          type: string
+      - name: germplasm
+        in: query
+        description: Filter by germplasm/cultivar.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+      - name: page
+        in: query
+        description: Which result page is requested. The page indexing starts at 0
+          (the first page is 'page'= 0). Default is `0`.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: integer
+        example: "0"
+      - name: pageSize
+        in: query
+        description: The size of the pages to be returned. Default is `1000`.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: integer
+        example: 1000
+      - name: Authorization
+        in: header
+        description: "HTTP HEADER - Token used for Authorization \n\n<strong>Bearer\
+          \ {token_string} </strong>"
+        required: false
+        style: simple
+        explode: false
+        schema:
+          pattern: ^Bearer .*$
+          type: string
+        example: Bearer XXXX
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/seasons'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: string
+              example: |-
+                ERROR - 2018-10-08T20:15:11Z - Malformed JSON Request Object
+                ERROR - 2018-10-08T20:15:11Z - Invalid query parameter
+                ERROR - 2018-10-08T20:15:11Z - Required parameter is missing
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - Missing or expired authorization
+                token
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - User does not have permission
+                to perform this action
+components:
+  schemas:
+    product:
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          example : 'Canopy Cover'
+    products:
+      type: array
+      items:
+        $ref: "#/components/schemas/product"
+    searchResult:
+      required:
+        - experiment
+      properties:
+        site:
+          type: string
+          example: Maricopa, AZ
+        observationunitname:
+          type: string
+          example: 'Row 1 Column 1'
+        experiment:
+          type: string
+          example: Drought Tolerance
+        germplasm:
+          type: string
+          example: Sorghum
```

Comment:
Sorghum is an example of a species. A germplasm is one of the 1000’s of varieties of Sorghum (or other species) used in our trials. Examples of germplasm should come from the BrAPI germplasm endpoint or BETYdb cultivars table.

Reply:
I used one from a brapi endpoint.

Reply:
From the first record returned by https://brapi.workbench.terraref.org/brapi/v1/germplasm, genus is Sorghum, commoncropname is 'sorghum' and 'germplasmName' is 'PI564163'

![image](https://user-images.githubusercontent.com/464871/60050806-85301c80-969f-11e9-95d6-e96971899e35.png)

Reply:
I created a scheme 'germplasm' with keys genus, commoncropname and germplasmName. An object with those keys will now be returned instead of just a string for the germplasmName in any API method.

Reply:
should be fixed. added germplasm schema to yaml file